config/locales/en.yml

```diff
@@ -28,6 +28,7 @@ en:
     finance: 'Billing API'
     stats: 'Analytics API'
     account_management: 'Account Management API'
+    policy_registry: 'Policy Registry API'
```

Comment:
Should this one be `policy_registry_api` too? Having the same local twice with diff values may be confusing.

Reply:
You are right there are many duplication, depending on the context though... I do no know if it is the right PR to refactor this

Reply:
Not in the PR. As you can see, the other entries for API group names do not have `_api` in the name of the key.